falconjs/templates/default/server.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-console */
```

Comment:
The file will be rewritten after merge falcon routing. For now the file do nothing.